src/components/Avatar/Avatar.vue

```diff
@@ -346,6 +385,26 @@ export default {
 			}
 			this.isMenuLoaded = true
 		},
+		async fetchUserStatus() {
+			if (!this.user) {
+				return
+			}
+			const user = encodeURIComponent(this.user)
+
+			try {
+				const { data } = await axios.get(generateOcsUrl('apps/user_status/api/v1', 2) + `statuses/${user}`)
```

Comment:
I think this should be cached ideally in a way that multiple component instances can access the status. I assume this could perform very bad if an avatar is rendered multiple times e.g. in a talk conversation.

Reply:
Ah missed your todo list for later :D